frontend/src/main/java/cz/cuni/mff/xrg/odcs/frontend/gui/MenuLayout.java

```diff
@@ -136,7 +146,17 @@ public void buttonClick(Button.ClickEvent event) {
                 authService.logout(RequestHolder.getRequest());
                 authCtx.clear();
                 refreshUserBar();
-                navigator.navigateTo(Login.class);
+                String logout_url = null;
+                try {
+                    logout_url = appConfig.getString(ConfigProperty.LOGOUT_URL);
```

Comment:
we have two logout URLs, one in the application and another in the cas.
spring security cas plugin ensures that if the local logout is called the
request is forwarded to the cas logout

On Sun, Mar 22, 2015 at 12:26 PM, Tomas Knap <notifications@github.com>
wrote:

> In
> frontend/src/main/java/cz/cuni/mff/xrg/odcs/frontend/gui/MenuLayout.java
> <https://github.com/UnifiedViews/Core/pull/282#discussion_r26901954>:
>
> > @@ -136,7 +146,17 @@ public void buttonClick(Button.ClickEvent event) {
> >                  authService.logout(RequestHolder.getRequest());
> >                  authCtx.clear();
> >                  refreshUserBar();
> > -                navigator.navigateTo(Login.class);
> > +                String logout_url = null;
> > +                try {
> > +                    logout_url = appConfig.getString(ConfigProperty.LOGOUT_URL);
>
> The ConfigPropety.LOGOUT_URL should be renamed to
> ConfigPropety.SSO_LOGOUT_URL, so that it is clear that for standalone use
> it is not needed
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/UnifiedViews/Core/pull/282/files#r26901954>.
>